.gitignore

```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
-
+/client/.env
```

Comment:
You actually don't need to gitignore this file, since any environment variables be exposed on the front-end anyways, and this makes it easier to share any configurations you have between each other

Reply:
Updated. Removed this from gitignore